### PR TITLE
IGN translator rewritten in Framework

### PR DESCRIPTION
**Pull request by owcz**
Rewritten in Framework, replacing antiquated, outdated version

### CODE REVIEW

File: IGN.js [hunk 1]
```diff
@@ -1,103 +1,105 @@
 {
-	"translatorID": "d210c5a1-73e1-41ad-a3c9-331d5a3ead48",
+	"translatorID": "cecafe6f-7fc9-4bdb-ae70-bd0a75a8ca52",
```
Comment (zuphilip) [inline IGN.js]: You should never change an existing `translatorID` for a translator, if you are updating it.

**Comment by zuphilip** [comment]
I haven't looked at the details, but usually I think that a normal translator would be preferred compared to one that is using the framework.

File: IGN.js (hunk 1 above)
Comment (owcz) [inline IGN.js]: Even if it completely replaces the former translator, without re-using any of it?

**Comment by owcz** [comment]
Why? This performs the same function but isn't broken and is easier for others to maintain

File: IGN.js (hunk 1 above)
Comment (adam3smith) [inline IGN.js]: yes -- otherwise you won't replace the old translator.

**Comment by avram** [comment]
Probably the wrong forum for this, but I see little reason to discourage Framework translators that work. Yes, they're more limited and can't be customized as much. But they can also lower the bar for contribution and help increase the total coverage that our translators have.

**Comment by adam3smith** [comment]
agree with @avram here: As long as we don't need any clean-up on a translator, I don't see why we shouldn't do it as FW.

File: IGN.js [hunk 2]
```diff
@@ -1,103 +1,105 @@
...
-	}
-	
-	// attach html
-	newItem.attachments.push({title:"IGN Article Snapshot", document:doc});
-	
-	newItem.complete();
-}
+FW.Scraper({
+itemType         : 'blogPost',
+detect           : FW.Url().match(/articles\/\d{4}/),
```
Comment (adam3smith) [inline IGN.js]: given that you only detect on ign.com/articles/, could you add /articles/ to the target (leave it in detect to be safe)

**Comment by adam3smith** [comment]
Thanks, looks good. FWIW, I think it makes it easier to review if you don't squash your commits, especially now that we can just squash on merging.

**Comment by owcz** [comment]
Didn't know you could do that—noted & thanks!

**Pull request merged**